hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -52,6 +58,44 @@ objects:
     source: pagerduty-operator-catalog
     sourceNamespace: pagerduty-operator
 
+- apiVersion: pagerduty.openshift.io/v1alpha1
+  kind: PagerDutyIntegration
+  metadata:
+    name: osd-scale-test
+    namespace: pagerduty-operator
+  spec:
+    acknowledgeTimeout: ${{ACKNOWLEDGE_TIMEOUT}}
+    resolveTimeout: ${{RESOLVE_TIMEOUT}}
+    escalationPolicy: ${{SCALE_TEST_ESCALATION_POLICY}}
+    servicePrefix: ${{SCALE_TEST_SERVICE_PREFIX}}
+    pagerdutyApiKeySecretRef:
+      name: pagerduty-api-key
+      namespace: pagerduty-operator
+    clusterDeploymentSelector:
+      matchExpressions:
+      # only create PD service for managed (OSD) clusters
+      - key: api.openshift.com/managed
+        operator: In
+        values: ["true"]
+      # select specific organizations
+      - key: api.openshift.com/legal-entity-id
+        operator: In
+        values: ${{SCALE_TEST_LEGALENTITY_IDS}}
```

Comment:
This also needs excluded from the main PDI, else you get 2 secrets managed by hive named the same thing and.. Bad Things.

Reply:
Added the `SCALE_TEST_LEGALENTITY_IDS` selector to each PDI